test/modules/standard/Random/deitz/test1D2D.chpl

```diff
@@ -113,8 +127,10 @@ config const n: int = 8;
   var A: [{1..n} dmapped Block(rank=1,boundingBox={1..n/2})] real;
   forall (r,a) in zip(rs.iterate(A.domain), A) do
     a = r;
-  writeln(for e in A do format("#######", e.locale.id));
-  writeln(for e in A do format("#.#####", e));
+  for e in A do writef(" #######", e.locale.id);
+  writeln();
+  for e in A do writef(" #.#####", e);
+  writeln();
```

Comment:
Now that the other thread has convinced me that format() ("create a string from nothing") has value beyond swritef()/string.writef() (if we can even support that), do you think I should revert this test?  What about the testformat*.chpl tests?